### PR TITLE
PR feature/03--like-water

### PR DESCRIPTION
**Pull request by mfees**
initial pr

### CODE REVIEW

**Comment by mfees** [comment]
https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/44

**Comment by TaylorFoster90** [comment]
:+1:

**Comment by sak55** [comment]
:+1:

File: pub/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/main.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <header>
+            <img style="width:100%;" src="images/banner.jpg">
+        </header>
+        <div style="width:1060px; margin: 0 auto;">
```
Comment (al-the-x) [inline pub/index.html:21]: So, by _embedded styles_ what I meant were CSS rules and properties contained in a `<style>` tag. These are _inline styles_, CSS properties applied directly to an element with the HTML `style` property. There are important differences. Mostly, as you've pointed out, inline styles are really annoying.

File: pub/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                </div>
+
+                <form>
+                    <h2 style= "color:#bbb; font-size:14px; margin-bottom:18px; font-family:'Open-sans' sans-serif; text-transform:uppercase; font-weight:bold;">Search</h2>
+                    <input style="width: 100%; height: 25px; border-radius: 2px; border: solid 1px #e8e8e8;" type="text">
+                </form>
+            </aside>
+            <section style="float:left; width:75%; box-sizing:border-box; margin-top:62px;">
+                <article style="border-bottom: solid 2px #e8e8e8; padding-bottom:38px;">
+                    <h1 style="color:#39474e; font-size:40px; font-family:'Open-sans' sans-serif; font-weight:bolder;">My Boyfriend</h1>
```
Comment (al-the-x) [inline pub/index.html:36]: Really only one `<h1>` per page is preferred...

File: pub/index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                    <blockquote style="display:block; margin-top:32px;">
+                        <p style="font-size:17px; font-family:'PT serif' serif; color:#bbb; line-height:26px; font-style:italic; border-left:solid 2px #e8e8e8; padding-left:15px;"><em>Maddie and Kolby</em></p>
+                    </blockquote>
```
Comment (al-the-x) [inline pub/index.html:39]: Better as an `<address>` tag, but don't tell Susanna... She's ecstatic about writing blog posts as it is.

File: pub/index.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                    <p style="font-size:17px; font-family:'PT serif' serif; color:#666666; line-height:26px;">Erin is like a sister to me. She is my rock and I don't know what I would do without her. She has been there for some very hard time in my life and she constantly picks me up. She is my best friend for now and for always.</p>
+                    <p style="margin-top:32px; max-width:500px;"><img style="width:100%;"src="images/family1.jpg"></p>
+                    <p style="margin-top:32px; max-width:500px;"><img style="width:100%;"src="images/family2.jpg"></p>
```
Comment (al-the-x) [inline pub/index.html:55]: Better wrapped in a `<figure>` than a `<p>` tag, as the latter is reserved for paragraphs of _text_...

**Comment by al-the-x** [comment]
Honestly, I'd like to see more commits and more frequently. The finished product looks pretty good, even if you did take expert mode with the in-line styles, but giving yourself more frequent checkpoints will become essential in the larger projects.

**Pull request closed without merging**